### ISSUE TITLE
Move non-removable items out of Bookmarks

### ISSUE THREAD

**Issue by cassidyjames** [elementary/files#1614]
Brought up at https://github.com/elementary/files/discussions/1613 but is a smaller, more actionable request that I think would improve the distinction between static locations and user-manageable Bookmarks.

For example:

|  |
| ---- |
| Home |
| Recent |
|Trash |
| **Bookmarks** ▾ |
| Music |
| Pictures |
| Videos |
| Downloads |
| **Devices** ▸ |
| **Network** ▸ |

**Comment by jeremypw** [comment]
@danirabbit Is this something that you still want to progress?  I could include it in the Gtk4 port of the sidebar if desired.

**Comment by danirabbit** [comment]
Yeah I think this still makes sense and I'm guessing it would probably make your life easier to make the DND area clearer :sweat_smile:
